code/index.html

```diff
@@ -20,28 +22,51 @@
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1>Does the person have...</h1>
 
       <select id="questions">
+        <option value="" selected disabled>Select attribute</option>
```

Comment:
This is a nice addition and avoids the problem of not being able to choose the default option :)

Reply:
Emma helped me with this one and she sad that she got the tip from you so thanks! :D